scripts/build/llvm.sh

```diff
@@ -187,7 +187,7 @@ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
 elif [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
   # We use our own local cache if possible
   set +e
-  brew install "$HOME/Library/Caches/Homebrew/llvm\@${LLVM_VERSION}*.bottle.tar.gz"
+  brew install $HOME/Library/Caches/Homebrew/llvm\@${LLVM_VERSION}*.bottle.tar.gz
```

Comment:
This fails with spaces in `$HOME`. Does
```bash
brew install "$HOME/Library/Caches/Homebrew/llvm\@${LLVM_VERSION}"*".bottle.tar.gz"
```
work?

Reply:
@251 Nope, that doesn't work - already tried.
But I retried with `$HOME` only quoted.

Reply:
What? Ancient Bash on OSX or what is the issue?